.github/workflows/updatecli-compose.yml

```diff
@@ -0,0 +1,38 @@
+---
+name: updatecli-compose
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 6 * * *'
+
+permissions:
+  contents: read
+
+jobs:
+  compose:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+      packages: read
+      pull-requests: write
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: docker/login-action@0d4c9c5ea7693da7b068278f7b52bda2a190a446 # v3.2.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
```

Comment:
Should this be a bot instead? It seems that [actor](https://github.com/orgs/community/discussions/47855) is not well defined, and would run in the context of an employee, which doesn't seem to be ideal.

Reply:
Interesting. The docs explained how to use this approach:
- https://github.com/docker/login-action#github-container-registry
- https://docs.github.com/en/packages/managing-github-packages-using-github-actions-workflows/publishing-and-installing-a-package-with-github-actions#upgrading-a-workflow-that-accesses-ghcrio

Maybe it's not the case for this specific use case. We have used this approach any issues at all:
- https://github.com/elastic/apm-server/actions/runs/10935341957/job/30356922711#step:4:1

I can see my username, but the following steps do what's expected (download the containers):

<img width="825" alt="image" src="https://github.com/user-attachments/assets/f0911669-f0ff-43f7-83b7-aee57054a0e1">


Likely GitHub does something special with the login and it's not honoured but the GITHUB_TOKEN.

So far I think we are safe

Reply:
Thanks for the links. I think you're correct that the `actor` is not honored and the `GITHUB_TOKEN` takes precedence. It's just a bit strange.